### PR TITLE
chore: create Rust release runbook

### PR DESCRIPTION
**Pull request by ajewellamz**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: DynamoDbEncryption/runtimes/rust/test_examples/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+edition = "2021"
+rust-version = "1.80.0"
+keywords = ["cryptography", "security", "dynamodb", "encryption", "client-side"]
+license = "ISC AND (Apache-2.0 OR ISC)"
+description = "aws-db-esdk is a library for implementing client side encryption with DynamoDB."
+homepage = "https://github.com/aws/aws-database-encryption-sdk-dynamodb/tree/main/releases/rust/db_esdk"
+repository = "https://github.com/aws/aws-database-encryption-sdk-dynamodb/tree/main/releases/rust/db_esdk"
+authors = ["AWS-CryptoTools"]
+documentation = "https://docs.rs/crate/aws-db-esdk"
+readme = "README.md"
```
Comment (RitvikKapila) [inline DynamoDbEncryption/runtimes/rust/test_examples/Cargo.toml:13]: Are we going to publish this? If not, can we clean this similar to [Cargo.toml in TestVectors](https://github.com/aws/aws-database-encryption-sdk-dynamodb/blob/ajewell/release-scripts/TestVectors/runtimes/rust/Cargo.toml)?

File: DynamoDbEncryption/runtimes/rust/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+To publish a new version of the aws-db-esdk, version 1.2.3
+
+1. Acquire the appropriate permissions
+1. Ensure git checkout of main is fresh and clean
+1. ./start_release.sh 1.2.3
+1. cd ../../../releases/rust/db_esdk
+1. Create a PR with changes
+1. cargo publish
+1. cd ../../../DynamoDbEncryption/runtimes/rust/ # i.e. return here
```
Comment (RitvikKapila) [inline DynamoDbEncryption/runtimes/rust/RELEASE.md:9]: Can we add a step to verify that the release is actually on cargo with the correct version?

File: DynamoDbEncryption/runtimes/rust/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+To publish a new version of the aws-db-esdk, version 1.2.3
+
+1. Acquire the appropriate permissions
+1. Ensure git checkout of main is fresh and clean
+1. ./start_release.sh 1.2.3
+1. cd ../../../releases/rust/db_esdk
+1. Create a PR with changes
```
Comment (RitvikKapila) [inline DynamoDbEncryption/runtimes/rust/RELEASE.md:7]: Can we add a little more details about what all is required in the PR?

**Review by RitvikKapila — COMMENTED** [review]
Added a few small comments

File: DynamoDbEncryption/runtimes/rust/RELEASE.md (hunk 2 above)
Comment (ajewellamz) [reply DynamoDbEncryption/runtimes/rust/RELEASE.md:9]: added into test_publised.sh

**Review by RitvikKapila — APPROVED** [review]
LGTM

**Pull request merged**